lightning/src/ln/channelmanager.rs

```diff
@@ -291,6 +291,28 @@ struct ReceiveError {
 	msg: &'static str,
 }
 
+/// This enum is used to specify which error data to send to peers when failing back an HTLC
+/// using [`ChannelManager::fail_htlc_backwards_with_reason`] in response to
+/// [`events::Event::PaymentClaimable`].
+/// 
+/// For more info on failure codes, see <https://github.com/lightning/bolts/blob/master/04-onion-routing.md#failure-messages>
+/// (enum variant descriptions are from here).
+#[derive(Clone, Copy)]
+pub enum FailureCode {
+	/// General temporary failure of the processing node.
+	TemporaryNodeFailure             = 0x2000 | 2,
+	/// The processing node has a required feature which was not in this onion.
```

Comment:
Just tried to reword and add some examples/elaboration here. Let me know if there's anything else I should include:)

Reply:
> also maybe say "us" for "the processing node"

@TheBlueMatt I think using "you" instead of "us" would be more consistent with our other docs for these failure codes, will be reviewing accordingly